core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java

```diff
@@ -141,7 +150,8 @@ ServiceConfig build(ServiceNaming defaultServiceNaming,
                 maxRequestLength != null ? maxRequestLength : defaultMaxRequestLength,
                 verboseResponses != null ? verboseResponses : defaultVerboseResponses,
                 accessLogWriter != null ? accessLogWriter : defaultAccessLogWriter,
-                accessLogWriter != null ? shutdownAccessLogWriterOnStop : defaultShutdownAccessLogWriterOnStop);
+                accessLogWriter != null ? shutdownAccessLogWriterOnStop : defaultShutdownAccessLogWriterOnStop,
+                blockingTaskExecutor);
```

Comment:
We could add the `defaultBlockingTaskExecutor` parameter to this method and pass non-null here.